argparse/__init__.py

```diff
@@ -155,11 +155,11 @@ def _parse_args_galaxy_argp(self, argp):
             version = '1.0'
         tool = gxt.Tool(
                 argp.prog,
-                argp.prog,
+                argp.prog.replace(" ", "_"),
                 version,
                 argp.description,
-                argp.prog,
-                interpreter='python',
+                "python "+argp.prog,
+                interpreter=None,
```

Comment:
yeah this was archaic. This change should've been made a while ago. We should probably auto-annotate the requirements to include python, but that's a separate issue.